test/extended/util/image/image.go

```diff
@@ -19,6 +21,14 @@ var (
 	fromRepository     string
 	images             map[string]string
 
+	releasePullSpecInitializationLock sync.RWMutex
+	releasePullSpecInitialized        bool
+	releasePullSpecRemoteMirror       = map[string]string{}
+	allowedReleasePullSpec            = map[string]string{
```

Comment:
another way to think about these pullspecs is that they are the pullspec to use if you want to pull the image using "pullthrough" via the internal cluster registry(which may save some time/cost over pulling them remotely)

that option only exists when:
1) the internal registry is running (obviously)
2) there exists an imagestreamtag representing the image that you can use as the pullthrough pullspec.

i'm not sure if it warrants renaming the variable (i might have called it "availablePullThroughPullSpecs" or "availablePullThroughImages"), but a comment explaining the distinction would be good.

it's not so much about it being "allowed" as it being available/possible/valid.

Reply:
Thanks for the feedback, that makes more sense, I went ahead and changed the name to `availablePullSpecs`